<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reconstruction Setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="4">
    <widget class="QPushButton" name="output_button">
     <property name="text">
      <string>Save As</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="recon_comboBox"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="output_lineEdit"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="input_button">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="output_label">
     <property name="text">
      <string>Output path:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QPushButton" name="job_button">
     <property name="text">
      <string>Job Parameters</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QPushButton" name="recon_button">
     <property name="text">
      <string>Recon Parameters</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QComboBox" name="output_comboBox"/>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="input_lineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="run_label">
     <property name="text">
      <string>Run job on:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="recon_label">
     <property name="text">
      <string>Reconstruction:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="input_comboBox"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="input_label">
     <property name="text">
      <string>Input Dataset:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="run_comboBox"/>
   </item>
   <item row="4" column="3" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="run_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
